src/medication-dispense/medication-dispense.resource.tsx

```diff
@@ -135,7 +135,16 @@ export function initiateMedicationDispenseBody(
       whenHandedOver: null,
       dosageInstruction: [
         {
-          text: medicationRequest.dosageInstruction[0].text,
```

Comment:
This is sort of ugly, I'd imagine there might be a better way to do this, but if it works, that's fine I guess.

Reply:
It's definitely more than sort of ugly, agreed.... I was hoping someone might have a "duh" better way to do it, but I can also take another stab before merging.

Reply:
Made this simpler.